content/200-concepts/100-components/drivers.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: 'Drivers'
```

Comment:
How do you feel about renaming to "Database Drivers" (both title, url and first headline in line 8)?

Reply:
No strong opinions. I think it makes sense ¯\_(ツ)_/¯

Reply:
I think the title and url make sense. I'm not so sure about the first headline yet. 🤔

Reply:
Follow-up thought: I don't think we should rename the first headline since it describes exactly what the section is about. "Database drivers" would not cover JS database drivers and serverless database drivers.